Nos/Views/NoteComposer/NoteUITextViewRepresentable.swift

```diff
@@ -47,6 +47,10 @@ extension NoteTextEditor {
                 .foregroundColor: UIColor.primaryTxt
             ]
             
+            if ProcessInfo.processInfo.isiOSAppOnMac {
```

Comment:
HOW DID YOU FIND THIS API? I HAVE GOOGLED FOR THIS SO MANY TIMES. I AM ANGRY AND ALSO VERY EXCITED.

Reply:
Whoa, what??? I've never found this, either! I thought there was no way to detect this! 😲🎉‼️

Reply:
Haha sooo much googling. I thought for sure there would be a compiler directive, but it makes sense you have to check at runtime I guess.